ia/src/protocole.py

```
@@ -3,7 +3,9 @@
 import game
 import ml_agent
 import random
+import time
```

Pylint (3.3.7)

[convention] standard import "time" should be placed before third party imports "game", "ml_agent"

(C0411)